docs/conversion_examples_gallery/index.rst

```diff
@@ -16,6 +16,7 @@ Recording
     :maxdepth: 1
 
     AlphaOmega <recording/alphaomega>
+    Axon <recording/axon>
```

Comment:
We also need to add this to the API docs.

Reply:
Good catch, thanks. Added. I also added Femtonics which was missing.